Source/WebKit/WebProcess/GPU/media/TextTrackPrivateRemote.cpp

```diff
@@ -62,22 +63,29 @@ void TextTrackPrivateRemote::updateConfiguration(TextTrackPrivateRemoteConfigura
 {
     if (configuration.trackId != m_id) {
         m_id = configuration.trackId;
-        if (client())
-            client()->idChanged(m_id);
+        notifyClients([id = crossThreadCopy(m_id)](auto& client) {
```

Comment:
See how you're cross-copying here. Might be fine but the mismatch made me worry.

Reply:
ok, it's a mistake to call callThreadCopy; copy/paste